js/classes/asset_manager.js

```diff
@@ -183,67 +183,67 @@ class AssetManager {
      */
     static get BAREBONES_DL_Q() {
         return [
+            // Entities:
+            Block.SPRITESHEET,
+            Chad.SPRITESHEET,
+            Crosshair.SPRITESHEET,
+            DialogBubble.SPRITESHEET,
+            OverheadIcon.SPRITESHEET,
+            PapaChad.SPRITESHEET,
+            Projectile.SPRITESHEET,
+            Slingshot.SPRITESHEET,
+            Sun.SPRITESHEET,
+            Sword.SPRITESHEET,
+            RuneDrop.SPRITESHEET,
+            RuneItem.SPRITESHEET,
```

Comment:
I think the `RuneDrop` and `RuneItem` spritesheets are (and will likely always be) the same, so we don't need both of these entries.